KubeArmor/BPF/shared.h

```diff
@@ -599,20 +715,26 @@ static inline int match_and_enforce_path_hooks(struct path *f_path, u32 id,
   return 0;
 
 ringbuf:
+
+  if(retval == DENY)
+    {
+      retval = -EPERM;
+    }
+  else 
+    retval = 0;
```

Comment:
Ensure correct formating
```suggestion
   retval = (retval == DENY)?-EPERM:0;
```